.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -1,40 +1,62 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Report a bug with the WordPress block editor or Gutenberg plugin
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+<!--
+Please fill out ALL sections. Bug reports with missing information will be
```

Comment:
In reality, we usually just ask the reporter for more information 🙂

Reply:
Nitpick - Should it say all required sections?

Could also make the 'ALL' even louder:
```
**ALL**
```

or 
```
<span style="font-size:99999999px">ALL</span>
```

Reply:
I think capitalising is enough. This is only shown when you're in the "Write" tab so any markdown and/or HTML won't be rendered.

```suggestion
Please fill out ALL required sections. Bug reports with missing information will be
```